ui/page/seedbackup/save_seed.go

```diff
@@ -126,6 +126,7 @@ func (pg *SaveSeedPage) OnNavigatedTo() {
 			seed, err := pg.wallet.DecryptSeed(password)
 			if err != nil {
 				m.SetError(err.Error())
+				m.ParentWindow().Reload()
```

Comment:
why we need reload in here? I see we don't have reason to do it

Reply:
It was added to fix the modal not refreshing after performing an action until somewhere on the screen is touched